src/main/java/seedu/duke/util/ExceptionMessage.java

```diff
@@ -21,44 +21,45 @@
             + "Perhaps you meant a different module?\n";
     public static final String MESSAGE_TASK_NOT_FOUND =
             "Sorry, that task index is not valid.\n"
-            + "Use list -t to look at your tasks, and their index.\n"
+            + "Use 'list -t' to look at your tasks, and their index.\n"
             + "You can't use the task index from the summary command!\n";
     public static final String MESSAGE_NO_EDIT_MODULE =
             "Please enter a proper module code to edit.\n"
             + "It should be a module code of an actual NUS mod. E.g. CS1010.\n";
     public static final String MESSAGE_NO_EDIT_TASK =
             "Please enter a proper task index to edit.\n"
-            + "Use list -t to look at your tasks, and their index.\n"
+            + "Use 'list -t' to look at your tasks, and their index.\n"
             + "You can't use the task index from the summary command!\n";
     public static final String MESSAGE_NO_ADD_MODULE =
             "Please enter a new module code to add.\n"
             + "It should be a module code of an actual NUS mod. E.g. CS1010.\n";
     public static final String MESSAGE_NO_ADD_TASK =
             "Please enter a new task to add.\n"
-            + "Use list -t to look at your tasks, and their index.\n"
+            + "Use 'list -t' to look at your tasks, and their index.\n"
             + "You can't use the task index from the summary command!\n";
     public static final String MESSAGE_INVALID_PARAMETERS =
-            "Some invalid or missing parameters were found!";
+            "Some invalid or missing parameters were found!\n"
+            + "Use 'help' to find out about the parameters required: 'help %s'\n";
     public static final String MESSAGE_LIST_EMPTY =
             "\nYour list is empty.\n";
     public static final String MESSAGE_STRING_IN_NUMBER =
             "The task index you have entered is either not an integer or is too long!\n"
-            + "Use `list -t` to look at your tasks, and their index.\n"
-            + "Please check your command! Use help <command> for assistance.\n";
+            + "Use 'list -t' to look at your tasks, and their index.\n"
+            + "Please check your command! Use 'help %s' for assistance.\n";
     public static final String MESSAGE_INVALID_COMMAND_WORD =
-            "I don't recognise that command...\n"
-            + "Please use help to see our list of commands.\n";
+            "ra.VI does not recognise that command.\n"
+            + "Please use 'help' to see our list of commands.\n";
     public static final String MESSAGE_INVALID_GRADE =
             "Your grade input isn't part of the NUS grading scheme. \n"
-            + "For your reference: A+, A, A-, B+, B, B-, C+, C, C-, D+, F. \n"
-            + "Grades CS, CU, are not accepted. \n";
+            + "For your reference: A+, A, A-, B+, B, B-, C+, C, C-, D+, F.\n"
+            + "Do not enter grades CS and CU - leave the module ungraded!\n";
```

Comment:
ok this will conflict with my side as my grade treats this as an exception, will edit this code portion later